src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -13,6 +12,7 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Collection;
 use Illuminate\Support\Str;
+use Symfony\Component\Console\Output\OutputInterface;
```

Comment:
The `symfony/console` dependency will need to be added to the component's composer.json file. It was already used in the artisan commands but those can only be used with the full framework anyway. Now that it's used in the Migrator class it needs to be an explicit dependency.

Reply:
Please note that we required on `Illuminate\Console\OutputStyle` before, while `illuminate/console` was not in the require section either. Yet, I've learned it's best/fasted to just follow requests from maintainers, so I've added `symfony/console` as a dependency (we can require any version, so I just picked all 4 and 5 versions)